apps/gp2-server/test/controllers/working-group.controller.test.ts

```diff
@@ -46,17 +73,37 @@ describe('Working Group controller', () => {
       workingGroupDataProviderMock.fetchById.mockResolvedValue(null);
 
       await expect(
-        workingGroupController.fetchById('not-found'),
+        workingGroupController.fetchById('not-found', '11'),
       ).rejects.toThrow(NotFoundError);
     });
 
     test('Should return the working group when it finds it', async () => {
       workingGroupDataProviderMock.fetchById.mockResolvedValue(
         getWorkingGroupDataObject(),
       );
-      const result = await workingGroupController.fetchById('working-group-id');
+      const result = await workingGroupController.fetchById(
+        'working-group-id',
+        '11',
+      );
 
       expect(result).toEqual(getWorkingGroupResponse());
     });
+
+    test('Should not return the resources when the user is not part of the team', async () => {
```

Comment:
same as above - add a member, just one that is not the same as the user who is fetching the data